### DISCUSSION TITLE
Running 'Enterprise-scale for small enterprises' Arm Deploy in Existing Footprint in Azure

### DISCUSSION THREAD

**Discussion by daz71** [Azure/Enterprise-Scale#568] [Q&A] [answered]
Hello,

We currently have an Azure tenancy and in it we run Azure Active Directory for our users that access Office 365 services. Would deploying the 'Enterprise-scale for small enterprises'(link below) in our tenancy impact on the current Azure AD or Office 365 services. Reading the documentation, it would seem that it won't as there's no mention of either.  

But grateful if anyone could also confirm before we deploy.

Thank you 

https://github.com/Azure/Enterprise-Scale/blob/main/docs/reference/treyresearch/README.md

**Comment by FallenHoot** [comment, marked as answer]
It shouldn't have any impact on your Azure AD or Office 365 users. Treyresearch will only deploy Management Group (Platform, Landing Zone, Sandbox) and depending on what options you pick during the UI screens, it will deploy Platform Subscription with management, connectivity, and identity.
